### DISCUSSION TITLE
igopher closes after launching issue

### DISCUSSION THREAD

**Discussion by devbob01** [hbollon/IGopher#26] [Q&A]
Hi , I really like what IGopher does, i wanted to try it but after I configure it with my IG account infos and the scrapped account, 
and I clicked on launch, it says that it is downloading depencies and then closes directly

**Comment by hbollon** [comment]
Hey! 
Glad you like IGopher 😁
Please open a dedicated [issue](https://github.com/hbollon/IGopher/issues/new/choose) with all required informations (log file, system specs, etc.) and I will try my best to help you.

**Reply by thacreatorr** [reply]
I have the same issue. I start it and then after 10 seconds or so the program shuts off and the screen closes
